.sync.yml

```diff
@@ -0,0 +1,61 @@
+---
+.travis.yml:
+  bundler_args: ' '
+  branches:
+    - release
+
+Gemfile:
+  required:
+    ':system_tests':
+      - gem: 'nokogiri'
```

Comment:
You may want to re-visit the location of some of these Gems, e.g. nokogiri would be probably more suited towards 'development'. Although it's not breaking tests now we don't use 'without system tests', this might be introduced back in the future and you could hit the same issue.

Reply:
@HelenCampbell moved gems to development.Modified .sync.yaml file